src/main/java/com/vispana/vespa/state/helpers/ProcessStatus.java

```diff
@@ -8,6 +8,10 @@
 public class ProcessStatus {
   public static Map<String, Status> processStatus(
       String host, Map<String, MetricsNode> vespaMetrics) {
+    if (!vespaMetrics.containsKey(host)) {
+      return Map.of();
```

Comment:
Is this just a safety check? Or is this a possible scenario with the new content topologies?